service/policy/namespaces/namespaces.go

```diff
@@ -93,12 +93,25 @@ func (ns NamespacesService) CreateNamespace(ctx context.Context, req *namespaces
 	}
 	rsp := &namespaces.CreateNamespaceResponse{}
 
-	n, err := ns.dbClient.CreateNamespace(ctx, req)
+	tx, err := ns.dbClient.BeginTx(ctx)
+	if err != nil {
+		ns.logger.Audit.PolicyCRUDFailure(ctx, auditParams)
+		return nil, db.StatusifyError(err, "begin txn failed", slog.String("name", req.GetName()))
+	}
+	// safe to ignore error per https://pkg.go.dev/github.com/jackc/pgx#hdr-Transactions
+	defer tx.Rollback(ctx)
```

Comment:
```suggestion
	//nolint noop https://pkg.go.dev/github.com/jackc/pgx#hdr-Transactions
	defer tx.Rollback(ctx)
```

Reply:
Alternatively so we don't have this cruft everywhere you might want to create a little util

```go
func txRollback(tx pgx.Tx, ctx context.Context) {
	//nolint noop https://pkg.go.dev/github.com/jackc/pgx#hdr-Transactions
  tx.Rollback(ctx)
}
```

Reply:
addressed in new commits